sycl/test-e2e/MemorySanitizer/lit.local.cfg

```diff
@@ -0,0 +1,12 @@
+# This test assumes it can only run in CPU/PVC/DG2 devices, which support usm_device_allocations aspect
```

Comment:
Unless you specify `REQUIRES:`, this assumption is incorrect. The test **will** be launched in other environments and on other devices as well, leading to unexpected and unrelated failures.

What you can do is just add `config.required_features += ['aspect-usm_device_allocations']` to this file to automatically set this requirement for all tests in sub-directory.

Reply:
This is line is not meaningful anymore, I removed it.